sms/receive-with-slim/composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "require": {
         "slim/slim": "^3.8",
-        "nexmo/client": "^2.0"
+        "vonage/client": "^2.0"
```

Comment:
Does 2.0 make sense, or should it be 2.2 when the vonage namespace was introduced?

Reply:
see https://github.com/Vonage/vonage-php-code-snippets/pull/50/files/32e99bfb89569773972f56405971b27a5c7b7a42#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780